lib/AllocateDescriptorsPass.cpp

```diff
@@ -26,9 +26,9 @@
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include "Passes.h"
```

Comment:
This (and other instances) no longer match the style of other local directory includes like ArgKind.h and Constants.h

Reply:
What do you mean?

Reply:
OK, fixed.  PTAL.